app/main/menu/submenu-items.js

```diff
@@ -140,8 +141,11 @@ items.reloadApp = {
   click(item, focusedWindow) {
     if (focusedWindow) {
       focusedWindow.webContents.send('reload-window');
-      focusedWindow.webContents.once('did-finish-load', () => {
-        focusedWindow.webContents.send('new-tab');
+
+      electron.ipcMain.once('guest-window-id', (event, id) => {
```

Comment:
I'be fixed this by having the main process decide on a window id, instead of the renderer process. This implies less ad-hoc communication between them.